src/components/ItemDetail/use/useOwners.ts

```diff
@@ -0,0 +1,30 @@
+import { ItemSummary } from '/@/lib/apis'
+
+export interface OwnerDetail {
+  userName: string
+  remain: number
+  all: number
+}
+
+const useOwners = (): {
+  getOwnerDetail: (props: { item: ItemSummary }) => OwnerDetail[]
+} => {
+  const getOwnerDetail = (props: { item: ItemSummary }) => {
+    return props.item.owners.map(owner => {
+      // そのownerの最後のログを取得
+      const latestLog = props.item.latestLogs?.find(
+        v => v.ownerId === owner.userId
+      )
+      const remain = latestLog ? latestLog.count : owner.count

```

Comment:
```suggestion
      const remain = latestLog?.count ?? owner.count
```
好みかもしれないけどこっちのがそれっぽそう